drivers/packet/option.go

```diff
@@ -56,7 +56,7 @@ func WithSSHKey(sshkey string) Option {
 }
 
 // WithTags returns an option to set the image.
-func WithTags(tags ...string) Option {
```

Comment:
The tags have been intentionally done like that :)

Reply:
thanks for the pointer , updated the PR 

`tags ...string` on this one what is the reason to format it like this?

Reply:
to call it like `packet.WithTags("a", "b", "c")` instead of `packet.WithTags([]string{"a", "b", "c"})`.

Reply:
For consistency with other drivers lets use `tags ...string` in this initial packet implementation. I am open to changing the function signature to `tags []string` but I would recommend handling this change in a separate pull request, and updating all drivers :)